extensions/qute/runtime/src/main/java/io/quarkus/qute/runtime/BuiltinExtensions.java

```diff
@@ -29,11 +30,17 @@ public static Object map(Map map, String name) {
                 return map.isEmpty();
             case "get":
                 return map.get(name);
```

Comment:
This one is a bit weird, isn't it? You're doing `map.get("get")` AFAICS?

Reply:
Good catch! This one should be removed...